alf/algorithms/generator.py

```diff
@@ -150,8 +149,7 @@ def __init__(self,
         self._net_moving_average_rate = net_moving_average_rate
         if net_moving_average_rate:
             self._predict_net = net.copy(name="Genrator_average")
-            tfa_common.soft_variables_update(
-                self._net.variables, self._predict_net.variables, tau=1.0)
+            common.get_target_updater(self._net, self._predict_net, tau=1.0)
```

Comment:
get_target_updater() only returns an updater. You need to call updater() to actually perform the update.
```python
self._predict_net_updater = common.get_target_updater(
   self._net,
   self._predict_net,
   tau=net_moving_average_rate)
```
At line252:
```python
self._predict_net_updater()
```

Reply:
corrected.